eventhub/Module.md

```diff
@@ -1,540 +1,199 @@
 
-## Azure Event Hubs Publisher Policy Operations
-The `ballerinax/azure_eventhub` module contains operations related to performing publisher policy operations on 
-event hubs. It includes operations to revoke publisher, get revoked publishers, resume upblisher.
+    1. Obtain the connection string for the Event Hub namespace by following the instructions given [here](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-get-connection-string#get-connection-string-from-the-portal).
+        (Eg: `"Endpoint=sb://<Namespace_Name>.servicebus.windows.net/;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=i1f2BXAbmtxhu............f9GMyfnNWvQ3CY="`)
 
-# Prerequisites:
+    2. Extract the shared access key name, shared access key, and resource URI of the Event Hub namespace from the connection string. 
+        * Shared access key name (Eg: `"RootManageSharedAccessKey"`)
+        * Shared access key (Eg: `"i1f2BXAbmtxhu............f9GMyfnNWvQ3CY="`)
+        * Resource URI to the Event Hub namespace (Eg: `"<Namespace_Name>.servicebus.windows.net"`)
 
-* Azure Account to Access Azure Portal https://docs.microsoft.com/en-us/learn/modules/create-an-azure-account/
+## Quickstart
 
-* A Resource Group https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create#create-a-resource-group
+To use the Azure Event Hub connector in your Ballerina application, update the .bal file as follows:
 
-* An Event Hubs Namespace https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create#create-an-event-hubs-namespace
-
-* An Event Hub https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create#create-an-event-hub 
-
-* Connection String of the Event Hub Namespace
-We need management credentials to communicate with the Event Hubs. These credentials are available in the connection 
-string of the Event Hub namespace. Obtain the connection string for the Event Hubs namespace by following the 
-instructions given below.
-https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-get-connection-string#get-connection-string-from-the-portal
-
-* Shared Access Signature (SAS) Authentication Credentials
-You need to extract the Shared Access Key Name, Shared Access Key, Resource URI to the Event Hub Namespace separately 
-from the connection string.
-    * Shared Access Key Name, 
-    * Shared Access Key, 
-    * Resource URI to the Event Hub Namespace.
-
-# Quickstart(s):
-
-## Publish Events to an Azure Event Hub 
-
-This is the simplest scenario to send events to an Azure Event Hub. You need to obtain a connection string of the 
-name space of the event hub you want to send events. 
-
-### Step 1: Import the Azure Event Hub Ballerina Library
-First, import the ballerinax/azure_eventhub module into the Ballerina project.
+### Step 1: Import connector
+First, import the `ballerinax/azure_eventhub` module into the Ballerina project.
```

Comment:
Let's remove the word "First"

Reply:
Resolved and amended the changes. Thank you for the suggestion.